tests/README.md

```diff
@@ -36,7 +38,7 @@ instead of running the tests.
 
 ## Runtime tests
 
-Runtime tests will call the bpftrace executable.
+Runtime tests will call the bpftrace executable. These are located in `tests/runtime` and are managed by a custom framework.
 
 * Run: `sudo make runtime-tests` inside your build folder
```

Comment:
Maybe also add that the tests can be executed by `sudo <builddir>/tests/runtime-tests.sh`?

Reply:
Good call. Done.